lib/generated_definitions/au.rb

```diff
@@ -92,8 +95,16 @@ def self.custom_methods
 end
 },
 
+"qld_kings_bday_october(year)" => Proc.new { |year|
```

Comment:
Are these procs even needed?  Surely could just set a before and after in defintions.  

All good though